server/files/utils/FeatureLexerFormatter.js

```diff
@@ -27,8 +27,8 @@ export default class FeatureLexerFormatter {
 
     feature (type, name, description) {
         let [inOrderTo, asA, iWant] = description.split(constants.FEATURE_NEWLINE);
-        inOrderTo = inOrderTo.replace(IN_ORDER_TO, '');
-        asA = asA.replace(AS_A, '');
+        inOrderTo = inOrderTo.replace(IN_ORDER_TO, '').replace(/\r/g, '');
```

Comment:
at the moment majority of people are on windows and this will solve there problem.We will make this make this compatible with mac  in later pull request. It is a short term solution but works.Let me know what you think.